README.md

```diff
@@ -26,23 +38,21 @@ on:
         type: string
         required: false
         description: "Version of yq to install, if not already"
-        default: "4.27.5"
-      debug:
-        type: boolean
+      mask-values:
```

Comment:
```suggestion
      mask-vars:
```
To match the action's input. I made the mistake of putting `mask-values` instead of `mask-vars` the first time I tried this out.